koku/masu/test/celery/test_tasks.py

```diff
@@ -147,23 +149,42 @@ def test_sync_data_to_customer_max_retry(self, mock_sync, mock_data_export_reque
     def test_get_currency_conversion_rates(self):
         with self.assertLogs("masu.celery.tasks", "INFO") as captured_logs:
             tasks.get_daily_currency_rates()
-            self.assertIn("Creating the exchange rate" or "Updating currency", str(captured_logs))
+
+        self.assertIn("Creating the exchange rate" or "Updating currency", str(captured_logs))
 
     # Check to see if Error is raised on wrong URL
     @patch("masu.celery.tasks.requests")
     def test_error_get_currency_conversion_rates(self, mock_requests):
-        mock_requests.get.side_effect = Exception("error")
-        with self.assertRaises(Exception) as e:
-            tasks.get_daily_currency_rates()
-            self.assertIn("Couldn't pull latest conversion rates", str(e.exception))
+        mock_session = Mock()
+        mock_session.get.side_effect = HTTPError("Raised intentionally")
+
+        mock_requests.Session.return_value = mock_session
+
+        with self.assertLogs("masu.celery.tasks", "ERROR") as captured_logs:
+            result = tasks.get_daily_currency_rates()
+
+        self.assertEqual({}, result)
+        self.assertIn("Couldn't pull latest conversion rates", captured_logs.output[0])
+        self.assertIn("Raised intentionally", captured_logs.output[1])
 
     @patch("masu.celery.tasks.requests")
     def test_get_currency_conversion_rates_successful(self, mock_requests):
         beforeRows = ExchangeRates.objects.count()
         self.assertEqual(beforeRows, 2)
-        mock_requests.get.return_value = Mock(
-            status_code=201, json=lambda: {"result": "success", "rates": {"AUD": 1.37, "CAD": 1.25, "CHF": 0.928}}
-        )
+
+        mock_response = Mock(status_code=201)
+        # fmt: off
+        mock_response.json.return_value = {
+            "result": "success",
+            "rates": {"AUD": 1.37, "CAD": 1.25, "CHF": 0.928}
+        }
+        # fmt: on
```

Comment:
Are these formatting commands necessary? Could we do:
```suggestion
        mock_response.json.return_value = {
            "result": "success",
            "rates": {"AUD": 1.37, "CAD": 1.25, "CHF": 0.928},
        }
```

Reply:
black was collapsing into one line since it's not quite as wide is line 200. I thought it was easier on the reader having the two test cases look visually similar, so I had to disable the formatting.

Reply:
Black is collapsing it because of the missing trailing comma after the rates dict (I think).

Reply:
That was it.